src/github3/checks.py

```diff
@@ -0,0 +1,416 @@
+        json = self._json(self._get(self.url), 200)
+        return self._instance_or_null(pulls.PullRequest, json)
+
+    refresh = to_pull
+
+
+class CheckApp(models.GitHubCore):
+    """Representation of an App returned in Checks APIs.
+
+    .. versionadded:: 1.3.0
+
+    .. note::
+
+        Refreshing this object returns a :class:`~github3.apps.App`.
+
+    This object has the following attributes:
+
+    .. attribute:: description
+
+        The description of the App provided by the owner.
+
+    .. attribute:: external_url
+
+        The URL provided for the App by the owner.
+
+    .. attribute:: html_url
+
+        The HTML URL provided for the App by the owner.
+
+    .. attribute:: id
+
+        The unique identifier for the App. This is useful in cases where you
+        may want to authenticate either as an App or as a specific
+        installation of an App.
+
+    .. attribute:: name
+
+        The display name of the App that the user sees.
+
+    .. attribute:: owner
+
+        A dict of minimal user information retrieved from the Check data
+        representing the app owner
+    """
+
+    def _update_attributes(self, app):
+        self.description = app['description']
+        self.external_url = app['external_url']
+        self.html_url = app['html_url']
+        self.id = app['id']
+        self.name = app['name']
+        self.owner = app['owner']
+
+    def _repr(self):
+        return '<App ["{}" by {}]>'.format(self.name,
+                                           str(self.owner['login']))
+
+    def to_app(self):
+        """Retrieve a full App object for this CheckApp.
+
+        :returns:
+            The full information about this App.
+        :rtype:
+            :class:`~github3.apps.App`
+        """
+        from . import apps
+        json = self._json(self._get(self.url), 200)
+        return self._instance_or_null(apps.App, json)
+
+    refresh = to_app
+
+
+class CheckSuite(models.GitHubCore):
+    """The :class:`CheckSuite <CheckSuite>` object.
+
+    .. versionadded:: 1.3.0
+
+    Please see GitHub's `CheckSuite Documentation`_ for more information.
+
+    .. attribute:: status
+
+        The status of the Check Suite
+
+    .. attribute:: conclusion
+
+        The highest priority check run conclusion. If it has not completed this
+        will be None
+
+    .. attribute:: head_sha
+
+        The sha of the commit at the head of the branch the check was run
+        against (the source of the pull request)
+
+    .. attribute:: head_branch
+
+        The branch checked
+
+    .. attribute:: before
+
+        The sha of the pull request target branch at the time of the checks
+
+    .. attribute:: after
+
+        The sha of the target branch after the change is applied
+
+    .. attribute:: repository
+
+        A representation of the repository the suite belongs to as
+        :class:`~github3.repos.repo.ShortRepository`.
+
+    .. attribute:: origional_pull_requests
```

Comment:
I think that should be `original_pull_requests` without the extra 'o'

Reply:
Good catch!